src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import PostgreSQLManagementClient from 'azure-arm-postgresql';
+import { DatabaseListResult, Server } from 'azure-arm-postgresql/lib/models';
+import { Databases } from 'azure-arm-postgresql/lib/operations';
+import * as vscode from 'vscode';
+import { AzureParentTreeItem, AzureTreeItem, ISubscriptionContext } from 'vscode-azureextensionui';
+import { getThemeAgnosticIconPath } from '../../constants';
+import { azureUtils } from '../../utils/azureUtils';
+import { PostgresDatabaseTreeItem } from './PostgresDatabaseTreeItem';
+import { PostgresSchemaTreeItem } from './PostgresSchemaTreeItem';
+import { PostgresTableTreeItem } from './PostgresTableTreeItem';
+
+export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionContext> {
+    public static contextValue: string = "postgresServer";
+    public readonly contextValue: string = PostgresServerTreeItem.contextValue;
+    public readonly childTypeLabel: string = "Database";
+    public readonly client: PostgreSQLManagementClient;
+    public readonly databases: Databases;
+    public readonly label: string;
+    public readonly server: Server;
+
+    constructor(parent: AzureParentTreeItem, client: PostgreSQLManagementClient, server: Server) {
+        super(parent);
+        this.client = client;
+        this.server = server;
+        this.label = server.name + ` (PostgreSQL)`;
```

Comment:
There's a separate `description` property that you should use for `PostgreSQL` instead of adding it to the label. It will handle the formatting for you

Reply:
Do you mean the "description" property for Experience interface or one from the AzureExtTree classes?  Also this label property for Cosmos accounts is created in SubscriptionTreeItem as such...
```
       const accountKindLabel = getExperienceLabel(databaseAccount);
       const label: string = databaseAccount.name + (accountKindLabel ? ` (${accountKindLabel})` : ``);
```
I am not sure I understand how to incorporate description to create this label.

Reply:
AzExtTreeItem class:

https://github.com/microsoft/vscode-azuretools/blob/master/ui/index.d.ts#L169

Reply:
`SubscriptionTreeItem` is probably doing it wrong. Here's a tree item that does it: https://github.com/microsoft/vscode-azureappservice/blob/138adfd6ff9dc65751a141c7b313c05968ff9127/src/explorer/SiteTreeItem.ts#L55

Reply:
Is description the same as the label that we see next to each account name in the tree view such "<Account name> (SQL)" ? I am not seeing any association to these two in given code..

Reply:
Yes - per [the docs](https://github.com/microsoft/vscode-azuretools/blob/master/ui/index.d.ts#L167)
> Additional information about a tree item that is appended to the label with the format `label (description)`

It also displays a little different in the treeItemPicker if `description` is used